pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -291,6 +296,7 @@ pub mod pallet {
 			currencies: BoundedVec<TokenMetaOf<T>, T::MaxCurrencies>,
 			denomination: u8,
 			transferable: bool,
+			min_operation_balance: u128,
```

Comment:
Why is this needed? Is it not enough to fail if the calculated collateral is `0`?

Reply:
Does the pallet go in an inconsistent state if these checks were omitted? Or what's the worst that can happen if the pool owner misconfigures this value?

Reply:
No, I don't think the pool goes into an inconsistent state. The relative error is just very high, but the minting and burning should still work. The user who wanted to mint the single unit can lose some money.

Reply:
Ok, then I don't have a strong opinion on this parameter.